### ISSUE TITLE
Buttons new functional colour variants

### ISSUE THREAD

**Issue by mccart77** [Orange-OpenSource/ods-android#153]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### As a

user of the demo app

### ... I want to

access the buttons component page

### ... so that I can

visualise the green and red functional buttons

### Motivation and context

New green and red functional buttons have been created and added to the DSM and Zeplin projects

### Proposed Solution

Add the new variants from Zeplin Android UI:
Contained - green and red functional
Contained with icon - green and red functional 

To the page:
Zeplin: Buttons 01 demo view

### Contact Details

stephen.mccarthy@orange.com

**Comment by afrahB** [comment]
Ok for the Enabled  Buttons.
KO for disabled  Buttons :  we need to increase the contrast of the text (disabled) against the background of the button

**Comment by afrahB** [comment]
there is the same issue for disabled buttons , the contrast of the text is not sufficient

**Comment by afrahB** [comment]
![Screenshot_20220909-103948_Accessibility Scanner](https://user-images.githubusercontent.com/103579735/189315508-0f62ff16-c9d4-49bc-8e88-3d663e4f03e2.jpg)

**Comment by afrahB** [comment]
![Screenshot_20220909-104001_Accessibility Scanner](https://user-images.githubusercontent.com/103579735/189315675-13430fca-a162-48dd-95f9-27c802f0c068.jpg)

**Comment by mccart77** [comment]
According to the W3C standards that I am aware of, disabled or inactive buttons do not need to meet any contrast ratio. Here is a quote from the page and the reference:

Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have **no contrast requirement.**

ref: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html 

Is this the standard that we are following, or is there another standard that you are using as your reference?

**Comment by afrahB** [comment]
Ok for the disabled components. you are right.
Just for me, I'm testing also the components from the application's perspective, to have usually a visible information 
so that all the components are accessible and to not affect the accessibility level of the app.
But it's ok for this case. We can't do it otherwise.
